src/viam/components/board/client.py

```diff
@@ -243,3 +249,29 @@ async def write_analog(
             extra = {}
         request = WriteAnalogRequest(name=self.name, pin=pin, value=value, extra=dict_to_struct(extra))
         await self.client.WriteAnalog(request, timeout=timeout)
+
+    async def stream_ticks(
+        self,
+        interrupts: List[Board.DigitalInterrupt],
+        *,
+        extra: Optional[Dict[str, Any]] = None,
+        **__,
+    ) -> Stream[Tick]:
```

Comment:
(minor) similar to above, should this be `TickStream`? If not, should we remove the `TickStream` typedef?